cypress/Dockerfile

```diff
@@ -10,8 +10,6 @@ RUN pip3 install boto3 requests
 RUN npm install --save-dev axe-core \
     cypress-cucumber-preprocessor cypress-plugin-tab
 
-# unset CI which gets set in Cypress's dockerfile
```

Comment:
FYI removed this as CI is now replaced by CYPRESS_CI so no longer a name collision